tgui/packages/tgui/interfaces/PersonalCrafting.tsx

```diff
@@ -832,7 +848,7 @@ const RecipeContent = ({ item, craftable, busy, mode, diet }) => {
             <Stack.Item pl={1}>
               {!item.non_craftable && (
                 <Button
-                  width="104px"
+                  width={item.mass_craftable ? '74px' : '104px'}
```

Comment:
You *should* be able to accomplish this without hard setting the width like this? By using `grow`. Granted it's a bit finicky so if it doesn't end up working it's whatever.

Reply:
grow is for flex, button is box. I couldn't find a good option, but if you can tell me, that would be cool